pkg/bindings/images/build.go

```diff
@@ -481,9 +481,9 @@ func nTar(excludes []string, sources ...string) (io.ReadCloser, error) {
 					return nil // skip root dir
 				}
 
-				name := strings.TrimPrefix(path, s+string(filepath.Separator))
+				name := filepath.ToSlash(strings.TrimPrefix(path, s+string(filepath.Separator)))
```

Comment:
What exactly is changing here?  I am not seeing it?

Reply:
AIUI [filepath.ToSlash()](https://pkg.go.dev/path/filepath#ToSlash), when run on Windows, will have `separator` set to backslash and will convert those to normal slashes.